Libraries/Text/RCTShadowText.m

```diff
@@ -119,7 +119,7 @@ - (NSTextStorage *)buildTextStorageForWidth:(CGFloat)width
 
   NSTextContainer *textContainer = [NSTextContainer new];
   textContainer.lineFragmentPadding = 0.0;
-  textContainer.lineBreakMode = _numberOfLines > 0 ? NSLineBreakByTruncatingTail : NSLineBreakByClipping;
```

Comment:
My only concern is this change of behavior. Previously, the default `lineBreakMode` was `NSLineBreakByTruncatingTail` or `NSLineBreakByClipping`, depending on the `numberOfLines`. Now it's `NSLineBreakByWordWrapping`.

I've no idea why the original defaults were chosen, or whether `NSLineBreakByWordWrapping` is a better choice, but this change has the potential to break all sorts of things. Can we find a way to implement this so that the defaults aren't affected?

A simple solution would be to add something like this in the Text.js render function (code not actually tested):

    if (Platform.OS === 'ios' && props.lineBreakMode === undefined) {
      props.lineBreakMode = props.numberOfLines > 0 ? 'truncating-tail' : 'clipping';
    }